src/no/runsafe/warpdrive/database/WarpRepository.java

```diff
@@ -101,22 +125,60 @@ public void DelAllPrivate(String world)
 		database.execute("DELETE FROM warpdrive_locations WHERE world=? AND public=?", world, false);
 	}
 
-	private String cacheKey(String creator, String name, boolean publicWarp)
+	/**
+	 * Updates players stored with the default UUID to the UUID they have stored in UserControl
+	 * Requires player_db from UserControl to be a table in MySQL
+	 */
+	public void UpdateUUIDs()

```

Comment:
Just move this queries to the schema manager, and it will be done once rather than on each startup..